src/code/z_player_lib.c

```diff
@@ -843,24 +843,23 @@ Gfx* gPlayerSheath14DLs[2 * PLAYER_FORM_MAX] = {
     gLinkHumanSheathEmptyDL,
 };
 
-Gfx* D_801C00AC[][2] = {
-    { object_link_child_DL_01DBC8, object_link_child_DL_01DBC8 },
-    { object_link_child_DL_01DC18, object_link_child_DL_01DC18 },
+Gfx* gPlayerShields[][2] = {
+    { gLinkHumanHylianShieldWithMtxDL, gLinkHumanHylianShieldWithMtxDL },
+    { gLinkHumanMirrorShieldWithMtxDL, gLinkHumanMirrorShieldWithMtxDL },
 };
 
-Gfx* D_801C00BC[][2] = {
-    { gLinkHumanSheathDL, gLinkHumanSheathDL },
-    { object_link_child_DL_01DB40, object_link_child_DL_01DB40 },
-    { object_link_child_DL_01DB60, object_link_child_DL_01DB60 },
+Gfx* gPlayerSheathedSwords[][2] = {
+    { gLinkHumanSheathedKokiriSwordDL, gLinkHumanSheathedKokiriSwordDL },
+    { gLinkHumanSheathedRazorSwordDL, gLinkHumanSheathedRazorSwordDL },
+    { gLinkHumanSheathedGildedSwordDL, gLinkHumanSheathedGildedSwordDL },
 };
 
-Gfx* D_801C00D4[][2] = {
-    { object_link_child_DL_017700, object_link_child_DL_017700 },
-    { object_link_child_DL_017338, object_link_child_DL_017338 },
-    { object_link_child_DL_016B80, object_link_child_DL_016B80 },
+Gfx* gPlayerSwordSheaths[][2] = {
+    { gLinkHumanKokiriSwordSheathDL, gLinkHumanKokiriSwordSheathDL },
+    { gLinkHumanRazorSwordSheathDL, gLinkHumanRazorSwordSheathDL },
+    { gLinkHumanGildedSwordSheathDL, gLinkHumanGildedSwordSheathDL },
 };
 
-// Possibly two-handed swords
 Gfx* gPlayerLeftHandTwoHandSwordDLs[2 * PLAYER_FORM_MAX] = {
```

Comment:
These arrays inconsistent now, some have variable names saying they're DLs and some don't.

Reply:
Yes, that's intentional. The ones with the `DLs` suffix are part of the model groups stuff (`sPlayerDListGroups`).
All of them have `2 * PLAYER_FORM_MAX` dlists in them, 2 per form, one for near and one for far, but for some reason there's no far and near dlists, only one for each one, so every dlist is repeated. This seems to be a leftover from OoT.
The other dlists without that suffix are used directly by functions

Reply:
Hmm, I still don't find that very clear.